#!/usr/bin/env python
PACKAGE = "ros_people_model"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("run_face_landmarks",   bool_t,   0, "Generate face landmarks for each detected face",  True)
gen.add("run_face_id",   bool_t,   0, "Generate a face id for each detected face",  True)
gen.add("run_face_emotions",   bool_t,   0, "Classify the emotions of each face",  True)
gen.add("run_face_emotions_snet",   bool_t,   0, "Classify the emotions of each face",  True)
gen.add("run_eye_state",   bool_t,   0, "Classify the eye state of each face",  True)

exit(gen.generate(PACKAGE, "vis_dlib_faces.py", "RosPeopleModel"))
